doc/Config_Tutorials/Pbo_Creation_Dependence_Includes.md

```diff
@@ -0,0 +1,116 @@
+<!--
+I reccomend viewing this document with the extension Markdown Preview Enhanced (https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced). Once installed to VS Code you can see a html preview by clicking the pane on the right side of VS Code that looks like an open book with a magnifying glass. This will greatly enchance readability.
+-->
+
+# Initialization and #Include
+<p>In <a href="./Texture_Swaps.md">Texture Swaps</a> we mentioned that when pulling from another mod you need to both initialize a class (sometimes called defining it), and also declare the mod it comes from as a dependacy for our .pbo. This document will cover both how to do that and how #include works so that you know where to initialize a class. Note that this meaning of define is not the same as #define which is something else entirely.</P>

```

Comment:
```suggestion
<p>In [Texture Swaps](./Texture_Swaps.md) we mentioned that when pulling from another mod you need to both initialize a class (sometimes called defining it), and also declare the mod it comes from as a dependacy for our .pbo. This document will cover both how to do that and how #include works so that you know where to initialize a class. Note that this meaning of define is not the same as #define which is something else entirely.</P>

```

This is how you do links in markdown.